db/migrations/20171207225726-create-slack-workspace.js

```diff
@@ -1,6 +1,7 @@
 
 module.exports = {
   up: (queryInterface, Sequelize) => {
+    return Promise.all([
```

Comment:
I assume the sequelize version change required these migration changes? Weird that they make a breaking change for a minor version bump, but maybe we were "holding it wrong" the entire time 😄

Reply:
Yes, exactly. The minor change caused our migrations to fail in CI. I think I mentioned this in one of my commits but forgot to mention in the in PR body.